### ISSUE TITLE
[New Example] Rhythm a musical game example

### ISSUE THREAD

**Issue by UlisesFreitas** [GDevelopApp/GDevelop-examples#81]
## Rhythm a musical game for GDevelop

Basic example of how to create a music game like Dance Dance Revolution or Friday Night Funkin, The camera follows the player whos singing, and the arrows appear to that player. When the song ends calculate the scores and decides who wins.
## Controls:
- AWSD for player 1
- LEFT,RIGHT,UP,DOWN for player 2
- Enter for start the game
- R - Restart the game anytime 

## Checklist

- [x] My game has a proper name in the game properties. 
- [x] My game package name behind with `com.example.`.
- [x] My game has all events unfolded.
- [x] I've added myself as the author in the game properties.
- [x] I've included a file called "README.md" with a description in proper English, explaining what this example is doing.
- [x] I confirm that this game and all of its resources can be integrated to this GitHub repository, distributed and MIT licensed.
- [x] I've cleaned unused resources.

## Game online
https://games.gdevelop-app.com/game-f4f5cb84-335b-4346-a132-e98b223dd974/index.html

## Game folder
[GD-Examples-Rhythm-1.0.0.zip](https://github.com/GDevelopApp/GDevelop-examples/files/7072350/GD-Examples-Rhythm-1.0.0.zip)

**Comment by Midhil457** [comment]
Hi, so this isn't a full review or something, this are some suggestions and bugs I found when I did a quick look. I might not have the time to do a full one, so if someone can, please do a review after me. The game itself was fun, I spent some time playing it just for fun.
- The game is stuck after you finish the game/get the score near 30. I don't think it's the score causing the problem, I think the end card is not shown.
- Adding a permentant force when after creating the arrow instead of subtracting the Y position might be a more "conventional" way. 
- There is an unused resource Rival-micro-5
- I found the camera logic a bit confusing. Won't Centering the camera to the middle at start, and center them to the player or rival when it's their turn be fine? I think there might be a reson you chose this, but I don't know
- The background layer seems to be empty
- Adding a comment with the controls will be useful, I at first thought the rival controls were broken not knowing the rival used wasd 😅 
:)

**Comment by UlisesFreitas** [comment]
Ok, Im going to check this out and update the project

**Comment by UlisesFreitas** [comment]
The game is stuck after you finish the game/get the score near 30. I don't think it's the score causing the problem, I think the end card is not shown.
**- Fixed it's working now**

Adding a permentant force when after creating the arrow instead of subtracting the Y position might be a more "conventional" way.
**-Changed to Force at angle, it seems to work even better**

There is an unused resource Rival-micro-5
 **- Fixed is one of the animations is added now.**

I found the camera logic a bit confusing. Won't Centering the camera to the middle at start, and center them to the player or rival when it's their turn be fine? I think there might be a reson you chose this, but I don't know
**- I removed completely the camera just for the example since is not the relevant thing here.**

The background layer seems to be empty
**-Removed**

Adding a comment with the controls will be useful, I at first thought the rival controls were broken not knowing the rival used wasd 😅
:)
**- Did you read the description of the game? Anyway I added 2 texts in game screen to show the controls.**

Here the updated version:
https://games.gdevelop-app.com/game-b3b1e91b-03af-42f0-9f1e-7c2baa584c11/index.html

Here the updated project: 1.0.1
[WTF-Wars-Rhythm-1.0.1.zip](https://github.com/GDevelopApp/GDevelop-examples/files/7073070/WTF-Wars-Rhythm-1.0.1.zip)

**Comment by Midhil457** [comment]
Hi, I got some time to take another look. I wasn't able to find much issues. I did make some small changes
- What I meant by adding a force was when it was created
![rhy-1](https://user-images.githubusercontent.com/73597906/131675787-d451aef0-a76f-4cf3-b532-f269f4c4afa0.png)

- Also changed the example name to just be `rhythm`. Hope you don't mind
- And some other tiny changs too

I should have opened a PR for this by the time I have commented this

Thank you for the contribution!
:)
